pages/stories/buildits-design-system-gravity/index.html.md

```diff
@@ -12,8 +12,8 @@ originalSourceSiteId: buildit-medium-blog
 blocks-njk:
   main: layouts/partials/articles/main.njk
 image:
-  src: images/articles/gravity.png
-  alt: Buildit's Gravity Design System
+  src: /images/articles/ibgds-title.jpg
```

Comment:
Images relative to our site should not begin with a forward slash `/`

Reply:
for clarity, this only applies to images which are used in nunjucks templates (ie, defined in frontmatter)